docs/Prowlarr/index.md

```diff
@@ -0,0 +1,47 @@
+# Prowlarr
+
+Here you will find a collection of tips for Prowlarr.
+
+*Prowlarr is an indexer manager/proxy built on the popular arr .net/reactjs base stack to integrate with your various PVR apps. Prowlarr supports management of both Torrent Trackers and Usenet Indexers. It integrates seamlessly with Lidarr, Mylar3, Radarr, Readarr, and Sonarr offering complete management of your indexers with no per app Indexer setup required (we do it all).*
+
+For Installation and Quick Start Guide please check the official [WikiArr](https://wiki.servarr.com/en/prowlarr){:target="_blank" rel="noopener noreferrer"}
+
+## Master
+
+Prowlarr does not yet have a stable release.
+
+## Develop
+
+![develop](https://img.shields.io/badge/dynamic/json?query=%24.version&url=https://raw.githubusercontent.com/hotio/prowlarr/testing/VERSION.json&label=Current%20Develop/Beta%20Version&style=for-the-badge&color=4051B5)
+
+(Beta): This is the testing edge. Released after tested in nightly to ensure no immediate issues. New features and bug fixes released here first after nightly. It can be considered semi-stable, but is still `beta`.
```

Comment:
Should be `Released after being tested in nightly`

Reply:
https://wiki.servarr.com/prowlarr/faq#how-do-i-update-prowlarr
![image](https://user-images.githubusercontent.com/6155095/196519722-11447c97-7dbd-4789-b42b-89860e6a6bd7.png)